homeassistant/components/lg_soundbar/config_flow.py

```diff
@@ -70,13 +86,18 @@ async def async_step_user(self, user_input=None):
         except ConnectionError:
             errors["base"] = "cannot_connect"
         else:
-            await self.async_set_unique_id(details["uuid"])
-            self._abort_if_unique_id_configured()
-            info = {
-                CONF_HOST: user_input[CONF_HOST],
-                CONF_PORT: DEFAULT_PORT,
-            }
-            return self.async_create_entry(title=details["name"], data=info)
+            if len(details) != 0:
+                unique_id = DOMAIN
```

Comment:
We should only allow this if there are no existing config entries for the integration.